src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Pipeline/HealthCheckTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NUnit.Framework;
+
+namespace SFA.DAS.EmployerIncentives.Api.AcceptanceTests.Pipeline
+{
+    [TestFixture]
+    [Parallelizable]
+    public class WhenCheckingPipeLine
+    {
+        [Test]
+        [Ignore("being reworked")]
```

Comment:
Why is this ignored? The test should have been updated to accommodate the rework

Reply:
oh because we are planning to rewrite the Acceptance tests in SpecFlow and these currently are in NUnit